src/radiation/source_terms_single_group.hpp

```diff
@@ -490,9 +497,9 @@ void RadSystem<problem_t>::AddSourceTermsSingleGroup(array_t &consVar, arraycons
 			}
 		} // end full-step iteration
 
-		AMREX_ASSERT_WITH_MESSAGE(ite < max_ite, "AddSourceTerms iteration failed to converge!");
+		AMREX_ASSERT_WITH_MESSAGE(ite < max_ite, "AddSourceTerms outer iteration failed to converge!");
 		if (ite >= max_ite) {
```

Comment:
I think it would be more conventional to call these variables `iter` and `max_iter` to indicate iteration counts.

Reply:
Done.